Xamarin.Forms.Core/TextElement.cs

```diff
@@ -6,6 +6,9 @@ static class TextElement
 			BindableProperty.Create(nameof(ITextElement.TextColor), typeof(Color), typeof(ITextElement), Color.Default,
 									propertyChanged: OnTextColorPropertyChanged);
 
+		public static readonly BindableProperty LetterSpacingProperty =
```

Comment:
what unit is letterspacing expressed in ? ratio, pixels, ...? if it's a measure, 0d is a good default, but not if it's a ratio

you probably need to declare a propertyChanged delegate, which pokes into the ITextTelement interface, and the implementation should InvalidateMeasure

Reply:
@StephaneDelcroix I added InvalidateMeasure as you requested.
The value is presented in units of "Pt". which converts depending on the platform.
0 is the default value, if the value is greater than 0 then the distance is greater and vice versa.